packages/client/mutations/UpdatePokerScopeMutation.ts

```diff
@@ -260,7 +276,30 @@ const UpdatePokerScopeMutation: StandardMutation<TUpdatePokerScopeMutation, Hand
         }
       })
     },
-    onCompleted,
+    onCompleted: (res, errors) => {
+      if (onCompleted) {
+        onCompleted(res, errors)
+      }
+      const {updatePokerScope} = res
+      const {meeting} = updatePokerScope
+      if (!meeting) return
+      const {viewerId} = atmosphere
+      const {meetingId, updates} = variables
+      const update = updates[0]!
+      const {service, action} = update
+      const searchQuery = getSearchQueryFromMeeting(meeting, service)
+      if (!searchQuery) return
+      const [queryString, searchQueryFilters] = searchQuery
+      SendClientSegmentEventMutation(atmosphere, 'Updated Poker Scope', {
```

Comment:
Select all functionality seems to be broken for me, and I don't see event in segment logs
<img width="780" alt="image" src="https://user-images.githubusercontent.com/466991/162723669-defc0bf5-2cb2-44a0-831f-253d4015e612.png">

https://user-images.githubusercontent.com/466991/162723931-ff59e6fb-6158-4245-9209-45a17cd1365d.mov

Reply:
Yeah, select all functionality is broken in production. I created an issue to fix it: https://github.com/ParabolInc/parabol/issues/6213

I'll look into why the event isn't firing

Reply:
@nickoferrall I think the issue is the following
```
Field "selectedAll" is not defined by type SegmentEventTrackOptions.
```